frb_codegen/src/generator/dart/ty_struct.rs

```diff
@@ -40,15 +40,17 @@ impl TypeDartGeneratorTrait for TypeStructRefGenerator<'_> {
             f.is_method_for_struct(&src.name) || f.is_static_method_for_struct(&src.name)
         });
         let has_methods = methods.next().is_some();
+        let access = "arr";
         let mut inner = s
             .fields
             .iter()
             .enumerate()
             .map(|(idx, field)| {
                 format!(
-                    "{}: _wire2api_{}(arr[{}]),",
+                    "{}: _wire2api_{}({}[{}]),",
                     field.name.dart_style(),
                     field.ty.safe_ident(),
+                    access,
```

Comment:
seems that (currently) the `access` variable is only used once (probably b/c some refactors), so there is no need to do code updates?